rotkehlchen/globaldb/handler.py

```diff
@@ -1197,7 +1197,29 @@ def get_manual_current_price(asset: Asset) -> Optional[Tuple[Asset, Price]]:
         return Asset(result[0]), deserialize_price(result[1])
 
     @staticmethod
-    def delete_manual_current_price(asset: Asset) -> None:
+    def get_all_manual_latest_prices(
+            from_asset: Optional[Asset] = None,
+            to_asset: Optional[Asset] = None,
+    ) -> List[Tuple[Asset, Asset, Price]]:
+        """Returns all the manual current prices in the price_history table"""
+        query = 'SELECT from_asset, to_asset, price FROM price_history WHERE source_type=?'
+        bindings = [HistoricalPriceOracle.MANUAL_CURRENT.serialize_for_db()]
+        if from_asset is not None:
+            query += ' AND from_asset=?'
+            bindings.append(from_asset.identifier)
+        if to_asset is not None:
+            query += ' AND to_asset=?'
+            bindings.append(to_asset.identifier)
+
+        with GlobalDBHandler().conn.read_ctx() as read_cursor:
+            result = read_cursor.execute(query, bindings)
+            return [
+                (entry[0], entry[1], deserialize_price(entry[2]))
```

Comment:
you do not need to do this at all. Just do a fetchall(). This is already 3 string values in each entry and this function is only used at one place, where you serialize to string to return through the api. So this is not needed at all